R/2-proprCall.R

```diff
@@ -177,31 +193,53 @@ updateCutoffs.propr <- function(object, cutoff = seq(.05, .95, .3)){
   FDR$cutoff <- cutoff
   p <- length(object@permutes)
 
-  # Calculate propr for each permutation -- NOTE: `select` and `subset` disable permutation testing
-  for(k in 1:p){
-
-    numTicks <- progress(k, p, numTicks)
-
-    # Calculate propr exactly based on @metric, @ivar, and @alpha
-    ct.k <- object@permutes[[k]]
-    pr.k <- suppressMessages(
-      propr(ct.k, object@metric, ivar = object@ivar, alpha = object@alpha))
-    pkt <- pr.k@results$propr
-
-    # Find number of permuted theta less than cutoff
-    for(cut in 1:nrow(FDR)){ # randcounts as cumsum
-
-      # Count positives as rho > cutoff, cor > cutoff, phi < cutoff, phs < cutoff
-      if(object@metric == "rho" | object@metric == "cor"){
-        FDR[cut, "randcounts"] <- FDR[cut, "randcounts"] + sum(pkt > FDR[cut, "cutoff"])
-      }else{ # phi & phs
-        FDR[cut, "randcounts"] <- FDR[cut, "randcounts"] + sum(pkt < FDR[cut, "cutoff"])
-      }
+  # Check for parallel package
+  parallelInstalled <- requireNamespace("parallel", quietly = TRUE)
```

Comment:
I just realized that `parallel` should be included with `R >= 2.14.0`.  So maybe this check isn't necessary?